core/commands/partials/terraform_configuration_generator.rb

```diff
@@ -200,6 +203,21 @@ def parse_products_info(node, cnf_template_path)
     products
   end
 
+  def generate_tags_list(tags = {})
```

Comment:
Most of the time you do not pass parameters to this method. Please consider to create a `configuration_tags` variable and pass it in all places you need this hash. In `generate_node_definition` method just simply extend the hash with the value of `configuration_tags`.